packages/main/src/Input.js

```diff
@@ -41,6 +41,8 @@ import styles from "./generated/themes/Input.css.js";
 import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverCommon.css.js";
 import ValueStateMessageCss from "./generated/themes/ValueStateMessage.css.js";
 
+const rgxFloat = new RegExp(/(\+|-)?\d+(\.|,)\d+/);
```

Comment:
Should we allow spaces? Or that's not valid for a number? In some countries they add a spce to delimit each thousands group.
There is also the scientific notation with the letter "E|e" also allowed, but probably not relevant.